config/initializers/productization.rb

```diff
@@ -1 +1,3 @@
-Vmdb::Productization.new.prepare
+Rails.application.config.to_prepare do
```

Comment:
Maybe something like this?  https://github.com/zohararad/wysihtml5n-rails/blob/1b73b2e21e2f800936c87b9e8032c46f21dd1f03/lib/wysihtml5n/rails/engine.rb#L4


![](https://media.giphy.com/media/ToMjGpBQn1JoHGuKjMA/giphy.gif)